src/soc/soc.h

```diff
@@ -53,7 +54,14 @@ typedef struct soc_t {
 	int (*validGPIO)(int);
 	int (*selectableFd)(int);
 	int (*gc)(void);
-
+	
+	int (*I2CSetup)(int);
```

Comment:
These are used anywhere in the main `wiringx.c` code.

Reply:
I also don't see the added functionality from these defines. Your I2C functions are the same as in `wiringx.c`